tests/sentry/notifications/utils/test_participants.py

```diff
@@ -331,10 +361,177 @@ def test_fallthrough(self):
 
     def test_without_fallthrough(self):
         ProjectOwnership.objects.get(project_id=self.project.id).update(fallthrough=False)
-        event = self.store_event("no_rule.cpp")
+        event = self.store_event_owners("no_rule.cpp")
 
         assert self.get_send_to_owners(event) == {}
 
+    @with_feature("organizations:streamline-targeting-context")
+    def test_send_to_suspect_committers(self):
+        self.release = self.create_release(project=self.project, version="v12")
```

Comment:
I may be wrong but I would stay away from setting any `self` properties because they may be non-None in following tests.

This is not an issue now but it will be the day we run tests in parallel or when tests are tested in a different order.

I'm also afraid that it may be the way that the super class has defined helper classes to work.

Ignore my comment in such case. It would have to be a project for another day.

Reply:
I agree with this, thanks for bringing it to my attention!